Jenkinsfile

```diff
@@ -1,3 +1,23 @@
+// Minor housekeeping logic
+boolean specialBranch = env.BRANCH_NAME.equals("master") || env.BRANCH_NAME.equals("develop")
+
+String[] jobNameParts = env.JOB_NAME.tokenize('/') as String[]
+String realPojrectName = jobNameParts.length < 2 ? env.JOB_NAME : jobNameParts[jobNameParts.length - 2]
+boolean originNanoware = jobNameParts[0].equals("Nanoware")
+boolean experimental = jobNameParts.length >= 2 && jobNameParts[2].startsWith("X")
```

Comment:
If the X here is meant to tie it to https://jenkins.terasology.io/teraorg/job/Nanoware/job/TerasologyModules/job/X/ maybe include a link to it? This could get confusing over time if things change and only part of the picture is readily visible :-)

Reply:
like so?